delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -369,7 +372,7 @@ def add_item(package=None):
         return item_list
 
     def _prepare_label_definition(self, picking):
-        error_missing = _(
+        error_missing = _lt(
```

Comment:
```suggestion
        error_missing = self.env._(
```

Reply:
Or `picking.env._` (since self is not a Record)

Reply:
Yes indeed. that will be an option.